src/Features/CSharpTest/Microsoft.CodeAnalysis.CSharp.Features.UnitTests.csproj

```diff
@@ -1,10 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
-<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetFramework>net472</TargetFramework>
```

Comment:
Not asking for this to be done in this PR - but this should allow us to target .net core for these tests right?  At some point I want to make that change and have them run on our core CI leg so we have coverage for non-windows platforms (now that this is all running in vscode).

Reply:
Yes, that'd be the ultimate goal. Unfortunately, it's not simple. Need to disentangle editor dependencies:

https://github.com/dotnet/roslyn/pull/71407